subsys/fs/Kconfig.littlefs

```diff
@@ -1,7 +1,15 @@
 # Copyright (c) 2019 Bolt Innovation Management, LLC
 # Copyright (c) 2019 Peter Bigot Consulting, LLC
+# Copyright (c) 2020 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
+config FILE_SYSTEM_LITTLEFS
+	bool "LittleFS support"
+	depends on FLASH_MAP
+	depends on FLASH_PAGE_LAYOUT
+	help
+	  Enables LittleFS file system support.
+
 menu "LittleFS Settings"
```

Comment:
Wonder if this and other should be wrapped with `if FILE_SYSTEM_LITTLEFS` (similar to how various `drivers/serial/Kconfig.*` files are structured).

Reply:
> 
> 
> Wonder if this and other should be wrapped with `if FILE_SYSTEM_LITTLEFS` (similar to how various `drivers/serial/Kconfig.*` files are structured).

The main `Kconfig` for `fs` does so, but if we are trying to be consistent across various subsystems, I may do the `if` for both the `Kconfig.littlefs` and `Kconfig.fatfs`.
I can not find an example within `drivers/serial/` that does so, can you point me out to such file?

Reply:
`drivers/serial/Kconfig.native_posix` was the example I looked at.